02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Ben's Site</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div>
+<img src="https://avatars2.githubusercontent.com/u/6634962?v=3&s=460"/>
+  </div>
+
+  <div>
+<h1 style="font-family: cursive">Benjamin Blowe</h1>
+  </div>
+<p>Born here in San Antonio, spent my childhood living in Florida, and have been back in SATX for the last decade. I'm an anthropology student at UTSA, I currently work in customer service, and I'm here at ACA to make a career in web development.</p>
+
+  <div>
+<h4>Contact:</h4>
+<h4>bdbblowe@gmail.com | (210) 487-8118</h4>
+</div>
```

Comment:
You can make the email hyperlink by <a href="mailto:me@anydomainname.com">eMail Us</a>
See: http://bit.ly/2ahxBCs
Good job!